db.py

```diff
@@ -1,21 +1,32 @@
 import sqlite3
+from peewee import SqliteDatabase
 import os
 
+# Initialize Peewee database
+db = SqliteDatabase('remo.db')
```

Comment:
When using SQLite in general, there are some pragmas, or options in SQLite, that you will want to enable. This mainly makes SQLite a bit more consistent. See `peewee` docs for the [recommended pragmas](https://docs.peewee-orm.com/en/latest/peewee/database.html#recommended-settings)

Reply:
Thanks for the heads-up.  Incoming change